### PR TITLE
Change: Refactor + adding tests

### PR DESCRIPTION
**Pull request by y0urself**
**What**:

**Why**:

**How**:

**Checklist**:

- [x] Tests
- [x] Conventional commit message
- [ ] Documentation

### CODE REVIEW

File: autohooks/plugins/isort/isort.py [hunk 1]
```diff
@@ -66,14 +80,20 @@ def precommit(config=None, **kwargs):  # pylint: disable=unused-argument
         ok('No staged files for isort available')
         return 0
 
+    arguments = ['black']
```
Comment (bjoernricks) [inline autohooks/plugins/isort/isort.py:83]: Shouldn't it be isort here?

**Review by bjoernricks — CHANGES_REQUESTED** [review]

File: pyproject.toml [hunk 2]
```diff
@@ -47,7 +47,6 @@ autohooks = ">=21.6.0"
 pontos = "^22.1.1"
 pytest = "*"
 pylint = "^2.12.2"
-isort = "^5.8.0"
```
Comment (bjoernricks) [inline pyproject.toml:50]: is this by intention?

**Review by bjoernricks — CHANGES_REQUESTED** [review]

File: autohooks/plugins/isort/isort.py (hunk 1 above)
Comment (y0urself) [reply autohooks/plugins/isort/isort.py:83]: Would be better, I guess!

File: pyproject.toml (hunk 2 above)
Comment (y0urself) [reply pyproject.toml:50]: Yes. If `isort` is already a dependencies it doesn't need to be a dev dependencies, too! :)

Comment (bjoernricks) [reply pyproject.toml:50]: Ah just removed from the devs section. Should have looked at the whole file :-)

**Review by bjoernricks — APPROVED** [review]

**Pull request merged**